detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/ForbiddenComment.kt

```diff
@@ -19,7 +19,7 @@ import org.jetbrains.kotlin.com.intellij.psi.PsiComment
  * </noncompliant>
  *
  * @configuration values - forbidden comment strings (default: `'TODO:,FIXME:,STOPSHIP:'`)
- *
+ * @configuration allowedPatterns - ignores comments which match any of specified regex (default: `""`)
```

Comment:
A more descriptive comment might help our users when configuring this rule in detekt.
@arturbosch are you okay with having multiple regular expressions in **one** config option?


```suggestion
 * @configuration allowedPatterns - ignores comments which match any of the specified regular expressions. 
 The regular expressions are separated by `,`, like for instance `\\s*Ticket\\s*, \\s*Task\\s*`. (default: `""`)
```

Reply:
Sure, will update the description once we decide if we are going with single config option or multiple.

Reply:
Updated the description as suggested

Reply:
Thanks! Would you mind also just changing "like for instance" to "for example"?

Will wait for Artur to confirm if multiple regex in one option is ok or not before merging. I've requested his review.

Reply:
I agree with @3flex 
This behavior would be very surprising for our users, since we only use a single regex as configuration options. We should be consistent here.
Furthermore, if a user configures a regex with comma `,`, this rule would break.

Reply:
> Furthermore, if a user configures a regex with comma `,`, this rule will break.

Yeah ! This is something that could potentially create problems. So, for a safer side, we can go with a single regex configuration and leave it to the user to support multiple expressions within it. Shall I go ahead and change it for wait for the existing code to be reviewed ?

Reply:
Yes, please change this to a single regex. This is the _detekt idiomatic_ way.
Everything else in this PR is fine. There’s no need to wait for another review.

Reply:
Thank you btw for the time to discuss this PR.